docs/source/getstarted.rst

```diff
@@ -38,6 +38,18 @@ Starting Up the SDK
 Example Programs
 ----------------
 
+^^^^^^^^^^^^^^^^^
+Walkthrough Video
+^^^^^^^^^^^^^^^^^
+
+For your convenience, here is a video detailing the first 8 simple example programs. There is also simple text-based walkthrough of running your first program below that.
```

Comment:
Maybe instead of "There is also simple text-based walkthrough of running your first program below that", you could say, "You can also find a simple text-based walkthrough of running your first program below" or similar.

Maybe writing 8 here isn't future proof? Don't want to have to remember to come back here. Or maybe it's fine.